migrator/migrations/m_171_to_m_172_move_scope_to_collection_in_report_configurations/accessScopePostgresStore/postgres_plugin.go

```diff
@@ -0,0 +1,282 @@
+package postgres
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/pkg/errors"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/logging"
+	ops "github.com/stackrox/rox/pkg/metrics"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	pkgSchema "github.com/stackrox/rox/pkg/postgres/schema"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/search/postgres"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+// This file is a partial copy of central/role/store/simpleaccessscope/postgres/store.go
+// in the state it had when the migration was written.
+// Only the relevant functions (UpsertMany, DeleteMany and Get) are kept.
+// The kept functions are stripped from the scoped access control checks.
+
+const (
+	baseTable = "simple_access_scopes"
+
+	batchAfter = 100
+
+	// using copyFrom, we may not even want to batch.  It would probably be simpler
+	// to deal with failures if we just sent it all.  Something to think about as we
+	// proceed and move into more e2e and larger performance testing
+	batchSize = 10000
+
+	deleteBatchSize = 5000
+)
+
+var (
+	log    = logging.LoggerForModule()
+	schema = pkgSchema.SimpleAccessScopesSchema
```

Comment:
It was my understanding that frozen schema is needed only if a migration is migrating objects from an old model to a new model. In this case, the already stored objects from old model would not match the new autogenerated schema and hence, we add a frozen schema for the older version before proto/model changes. Is frozen schema required for each table being read/written even if the change is not from an old model to a new model?

Reply:
Yes, if a migration needs to refer to any schema, it should be frozen. This is because the migration may exist a few versions while the schema in the master branch would evolve. In this case, you can create a copy of schema in the migration itself.

Reply:
Added all schemas to frozenschmas/v74